src/Compilers/Core/Portable/Symbols/ITypeSymbol.cs

```diff
@@ -111,6 +111,26 @@ public interface ITypeSymbol : INamespaceOrTypeSymbol
         /// has no concept of unmanaged types.
         /// </summary>
         bool IsUnmanagedType { get; }
+
+        // PROTOTYPE(nullable-api): Document
+        string ToDisplayString(Nullability topLevelNullability, SymbolDisplayFormat format = null);
+
+        // PROTOTYPE(nullable-api): Document
+        ImmutableArray<SymbolDisplayPart> ToDisplayParts(Nullability topLevelNullability, SymbolDisplayFormat format = null);
+
+        // PROTOTYPE(nullable-api): Document
+        string ToMinimalDisplayString(
+            SemanticModel semanticModel,
+            Nullability topLevelNullability,
+            int position,
+            SymbolDisplayFormat format = null);
+
+        // PROTOTYPE(nullable-api): Document
+        ImmutableArray<SymbolDisplayPart> ToMinimalDisplayParts(
+            SemanticModel semanticModel,
+            Nullability topLevelNullability,
```

Comment:
For these the semanticModel/position we usually treat as a pair and keep those together in the parameter ordering. So topLevelNullability should probably be either before or after that pair.